### DISCUSSION TITLE
How can I disable the directory completion hints on empty line

### DISCUSSION THREAD

**Discussion by chuan137** [marlonrichert/zsh-autocomplete#507] [Q&A] [answered]
I have the `auto_cd` option on, and same time I would like the shell to be a little cleaner on a empty line. Is it possible?
at the moment, the empty line shows sth like below

❯
 -- directory --
bin@       ca@        cc-shell/  certs/     git/     tmp/       venv@

**Comment by marlonrichert** [comment]
Put this in your `.zshrc` file, but change the value to `1`:
https://github.com/marlonrichert/zsh-autocomplete/blob/f07efda31f27446c8940b069d054bb8b0f61e026/.zshrc#L14

**Comment by jefftemplon** [comment]
Thanks for this!!  IMO this information should be promoted to the level of the main installation instructions - I was mystified by the combination of zsh-autocomplete and oh-my-zsh somehow turning this on whereas zsh-autocomplete minimal (per the bug report instructions) did not display the empty-line directory-completion hints.  Banged my head between the shell and google for an hour before finding this.

**Comment by marlonrichert** [comment, marked as answer]
Not showing dirs an empty command line is now the default behavior: 1af088b052f54511916709f1c6ca379ec4b79108

**Comment by jefftemplon** [comment]
Thanks!

**Reply by marlonrichert** [reply]
No problem! 😊

Oh, and if you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 💝
